controllers/drplacementcontrol_controller.go

```diff
@@ -243,6 +244,18 @@ func (r *DRPlacementControlReconciler) reconcileDRPCInstance(ctx context.Context
 		return ctrl.Result{}, err
 	}
 
+	// Currently validation of schedule in DRPolicy is done here. When
```

Comment:
Ack! @hatfieldbrian tagging you here for consideration once this and the initial DRPolicy reconciler are merged.

Reply:
DRPolicy reconciler is available in pull request #200.  Issue #201 tracks schedule validation move to DRPolicy reconciler.